src/utils/token-password.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable camelcase */
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+/**
+   * Hash Password Method
+   * @param {string} password
+   * @returns {string} returns hashed password
+   */
+const saltRounds = 10;
+const salt = bcrypt.genSaltSync(saltRounds);
+const hashPassword = password => bcrypt.hashSync(password, salt);
+
+/**
+   * comparePassword
+   * @param {string} hashPassword
+   * @param {string} password
+   * @returns {Boolean} return True or False
+   */
+const comparePassword = (hashedPassword, password) => {
+  return bcrypt.compareSync(password, hashedPassword);
+};
+
+/**
+   * Generate Token
+   * @param {string} id
+   * @returns {string} token
+   */
+const generateUserToken = ({ id, firstName, lastName, email }) => {
+  const token = jwt.sign({
```

Comment:
The `token` should be signed with only the `id`. Remove `firstName, lastName, email`

Reply:
ok @eokwukwe

Reply:
done @eokwukwe